src/request/mod.rs

```diff
@@ -43,7 +42,7 @@ pub struct Request<'a, 'b: 'a> {
     pub headers: Headers,
 
     /// The request body as a reader.
-    pub body: Body<'a, 'b>,
+    pub body: Cursor<Vec<u8>>,
```

Comment:
@MJDSys it is up to you if you want to continue this PR or leave it at that. No doubt it's a very useful start either way!